app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ClientServiceError < StandardError; end
```

Comment:
Custom errors must be defined in `app/lib/project_name/error.rb`. See: https://github.com/nimblehq/mia-web/blob/development/app/lib/mia/errors.rb

This allows to have an overview of all possible application errors in a single place/